platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -21,14 +21,40 @@
                 persistent-id="org.codice.ddf.security.policy.context.impl.PolicyManager"
                 update-strategy="component-managed" update-method="setPolicies"/>
 
-        <property name="authenticationTypes"
-                  value="/=SAML|ANON,/admin=SAML|basic,/jolokia=SAML|basic,/system=basic,/solr=SAML|PKI|basic,/sources=SAML|basic,/security-config=SAML|basic"/>
-        <property name="realms"
-                  value="/=karaf"/>
-        <property name="requiredAttributes"
-                  value="/=,/admin={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=admin},/solr={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=admin},/jolokia={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=admin},/system={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=admin},/sources={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=admin},/security-config={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=admin}"/>
-        <property name="whiteListContexts"
-                  value="${org.codice.ddf.system.rootContext}/SecurityTokenService,${org.codice.ddf.system.rootContext}/internal,/proxy"/>
+        <property name="authenticationTypes">
+            <array value-type="java.lang.String">
```

Comment:
At some point we need to re-evaluate our use of positive cardinality which uses primitive arrays compared to using negative cardinality which will inject vectors.